ago-dic-2018/Guadalupe Alejandro Pérez Camacho/practica_1/JavaApplication5.java

```diff
@@ -0,0 +1,48 @@
+package javaapplication5;
+import javax.swing.*;
+
+public class JavaApplication5 {
+    public static void main(String[] args) {
+        int n=Integer.parseInt(JOptionPane.showInputDialog(null,"N° de registros a ingresar"));
+        Alumno a[]=new Alumno[n];
+        int q=n;
+        String s="";
+        String ss="";
+        String Mayor_nombre="";
+        int Mayor_contador=0;
+        for (int i = 0; i < n; i++) {
+            boolean flag=false;
```

Comment:
Me revolvió un poco este for, pero en mi cabeza funciona, comoquiera en la clase lo corres para que lo veamos :+1:

Reply:
Con el ***for*** voy pidiendo los nombres de los alumnos y dentro de el se llena el arreglo de objetos, en el ***while*** recorro el arreglo hasta que se encuentre el nombre que se ingresó o se llegue a una casilla vacía, y ya con los ***if*** se aumenta el contador o se crea el registro